src/plugins/GUITestBase/src/GUITestBasePlugin.cpp

```diff
@@ -3094,6 +3095,11 @@ void GUITestBasePlugin::registerTests(UGUITestBase *guiTestBase) {
     REGISTER_TEST(GUITest_common_scenarios_workflow_name_filter::test_0002);
     REGISTER_TEST(GUITest_common_scenarios_workflow_name_filter::test_0003);
 
+    /////////////////////////////////////////////////////////////////////////
+    // Common scenarios/Workflow designer/Samples
+    /////////////////////////////////////////////////////////////////////////
+    REGISTER_TEST_TL(GUITest_common_scenarios_workflow_samples::test_0001, minutes(7), labels({Cistrome, Linux, Windows, MacOS}));
```

Comment:
The test itself looks simple, but you want to add to the special 'long-runing'  suite.
Unless we have a reason we should add tests to the Nightly (default) suite.
What is the execution time of the test?

Reply:
1 minute